### ISSUE TITLE
Disable WordPress Events and News Dashboard Widget

### ISSUE THREAD

**Issue by ipajen** [aspirepress/aspireupdate#172]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

I would like to have an option in AspireUpdate to disable the WordPress Events and News Dashboard Widget, similar to the functionality in the [disable-wordpress-events-and-news-dashboard-widget](https://github.com/dimadin/disable-wordpress-events-and-news-dashboard-widget) plugin. This setting would allow users to toggle the widget on or off, with the default set to  showing the widget (not ticked)

Option to add also a new Configuration Parameter for wp-config

The purpose of this feature is to prevent calls to WordPress.org, which may involve personal data. Until AspireCloud is equipped to manage these calls, having the option to disable them provides more control and avoids unnecessary content from official WordPress channels.

### Mockups or Images of the feature

new setting
![Settings](https://github.com/user-attachments/assets/17ab6a62-ff6d-4067-a2ce-2ff90b2f74f6)

before
![before](https://github.com/user-attachments/assets/f5589cd5-2b10-4b19-8867-cc9574e733d7)

after
![after](https://github.com/user-attachments/assets/786ff973-a2ba-4205-b81e-55649dc5b18f)

**Comment by asirota** [comment]
There are other plugins that allow these options to be set as you indicated. I think we need to be careful about turning on an option that calls w.org because that may change and we will need to catch up with WordPress team decision as to where/when Wordpress.org is called. This is better done as part of our telemetry documentation work to start identifying where w.org or api.org calls are made and explicitly letting customers who use AspirePress suite of software to determine what they would like to do.

Setting this at the AspireUpdate level is probably not the right place to do this. I would think this sort of policy should be better set at the AspireCloud level which would receive the telemetry information from WordPress installs.

**Comment by costdev** [comment]
I agree that the WordPress Events and News widget is in a different context to AspireUpdate, which is about installation and updates.

I think this would be better served by a plugin that either explicitly removes the widget, or better yet, filters its RSS sources with the existing filters in WordPress Core.

**Comment by namithj** [comment]
Closing as not planned.
